install.sh

```diff
@@ -20,26 +30,30 @@ gpasswd -a ${SUDO_USER:-$USER} epoptes
 echo 'IPAPPEND=3' >> /etc/ltsp/update-kernels.conf
 /usr/share/ltsp/update-kernels
 
-# configure dnsmasq
+# Configure dnsmasq
 ltsp-config dnsmasq
 
-# Creating lts.conf
-ltsp-config lts.conf
-
-# Creating client image
-ltsp-update-image --cleanup /
-
 # enabling password authentication 
 sed -i /etc/ssh/sshd_config \
-    -e "/Authentication no\$/ c PasswordAuthentication yes"
+    -e "/PasswordAuthentication no\$/ c PasswordAuthentication yes"
 service ssh restart
 
-# source setting.sh
-source /vagrant/settings.sh
+# Creating lts.conf
+lts_path=$(ltsp-config -o lts.conf|cut -d' ' -f2)
+
+# Client reboot issue fix (https://github.com/NetworkBlockDevice/nbd/issues/59)
+echo 'INIT_COMMAND_MV_NBD_CHECKUPDATE="mv /usr/share/ldm/rc.d/I01-nbd-checkupdate /usr/share/ldm/rc.d/I01-nbd-checkupdate.orig"' \
+    >> "$lts_path"
+
+# Installing additional software
+apt install --yes $PACKAGES
```

Comment:
What happens if `$PACKAGES` is empty?

Reply:
Nothing happens.

Reply:
No it does not. I can here is the output
```
default: + apt install --yes
    default: WARNING: 
    default: apt
    default:  
    default: does not have a stable CLI interface. 
    default: Use with caution in scripts.
    default: Reading package lists...
    default: Building dependency tree...
    default: Reading state information...
    default: 0 upgraded, 0 newly installed, 0 to remove and 39 not upgraded.

```